hello-world/app/js-vanilla/.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules
 wrap
+dist
 public/*.ts
-public/*.js
\ No newline at end of file
+public/*.js 
```

Comment:
This wasn't culprit, it was root level gitignore which was ignoring whole public dir